build-oracle-electron.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# downloads the bitcoin-s-server.zip file corresponding to your operating system 
+# then sets up Suredbits Wallets dependencies
+OS="`uname`"
+case $OS in
+  'Linux')
+   OS='Linux'
+    ;;
+  'WindowsNT')
+    OS='Windows'
+    ;;
+  'Darwin') 
+    OS='Mac'
+    ;;
+  'incompatible') ;;
+  *) ;;
+esac
+
+cd oracle-electron-ts
+
+Chip="`uname -m`"
+
+if [[ $Chip == 'arm64' ]]; then 
+	echo "you're on a M1 Mac need to generate bitcoin-s-oracle-server.zip if have not already done so"
+elif [[ $OS == 'Mac' ]]; then 
+	curl -O -L https://github.com/bitcoin-s/bitcoin-s/releases/download/1.9.2/bitcoin-s-oracle-server-mac.os.x-1.9.2.zip
+elif [[ $OS == 'Windows' ]]; then
+	curl -O -L https://github.com/bitcoin-s/bitcoin-s/releases/download/1.9.2/bitcoin-s-oracle-server-windows.server.2019-1.9.2.zip
+elif [[ $OS == 'Linux' ]]; then
+	curl -O -L https://github.com/bitcoin-s/bitcoin-s/releases/download/1.9.2/bitcoin-s-oracle-server-linux-1.9.2.zip
+
+else 
+	echo "not supported OS"
+fi
+
+for name in *bitcoin-s-server*; do
```

Comment:
for name in *bitcoin-s-oracle-server* ?

Reply:
You're right it should check for `bitcoin-s-oracle-server` for `build-oracle-electron.sh`. I don't think it will find any results in the`build-oracle-electron.sh`, the only reason it worked for me is because I accidentally put a `bitcoin-s-server` in my `oracle-electron-ts` directory. But it should be fixed now